### PR TITLE
Fix typos in the peek stage javadoc and signature

### PR DESCRIPTION
**Pull request by cescoffier**
Fix typos in the javadoc of the `peek` stage.

### CODE REVIEW

File: streams/api/src/main/java/org/eclipse/microprofile/reactive/streams/ProcessorBuilder.java [hunk 1]
```diff
@@ -66,14 +66,14 @@
...
+   * element.
    *
    * @param consumer The function called for every element.
-   * @return A new processor builder that consumes elements of type <code>T</code> and emits the same elements. In between, the given function 
-   * is called for each element.
+   * @return A new processor builder that consumes elements of type <code>T</code> and emits the same elements. In between,
+   * the given function is called for each element.
    */
-  public ProcessorBuilder<T, R> peek(Consumer<? super R> consumer) {
+  public ProcessorBuilder<T, T> peek(Consumer<? super T> consumer) {
```
Comment (jroper) [inline streams/api/src/main/java/org/eclipse/microprofile/reactive/streams/ProcessorBuilder.java]: I'm pretty sure this should be `<T, R>`, and `R` should be the peeked type. A processor takes in elements of type `T`, and emits elements of type `R`. This allows peeking on what is emitted, so the result is still something of type `R`.

Comment (cescoffier) [reply streams/api/src/main/java/org/eclipse/microprofile/reactive/streams/ProcessorBuilder.java]: Yes, you are right, I misread the generics.

**Comment by jroper** [comment]
This now has merge conflicts :/

**Comment by cescoffier** [comment]
Should be fixed now...

**Pull request merged**